### PR TITLE
docs(limitations): tracking + "when"

### PR DESCRIPTION
**Pull request by divyenduz**
Add tracking and "when" comments based on https://github.com/prisma/prisma2/issues/345

Notes: 
> Some edge cases for complex nested mutations don't work properly. 
- I don't think that this limitation exists anymore

### CODE REVIEW

**Comment by divyenduz** [comment]
From a user feedback: 

> here’s things that don’t appear to be:
- defined enums
- Json scalar type
- type definitions (or maybe just @connection.attributes?)
- inline embeds? I saw some stuff about regular embeds working in the latest beta, but I’m not sure if that also applies to inline ones

File: docs/limitations.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-- Does not work on Windows. Tested on Mac and Linux only.
-- Processes 1 request at a time, no parallelism during the early preview period.
-- Some edge cases for complex nested mutations don't work properly.
-- Limited auto-complete in Typescript projects due to a compiler bug.
-- No realtime API/subscriptions.
+- Does not work on Windows. Tested on Mac and Linux only. Tracking issue: https://github.com/prisma/prisma2/issues/4
+- Processes 1 request at a time, no parallelism during the early preview period. This will be fixed before GA. 
+- Some edge cases for complex nested mutations don't work properly. 
+- Limited auto-complete in Typescript projects due to a compiler bug. We raised a PR to fix that [here](https://github.com/microsoft/TypeScript/pull/32100). 
+- No realtime API/subscriptions. In future, we will have Prisma events engine (after the GA) that would pave way for such a feature, there is no ETA for this yet.
```
Comment (janpio) [inline docs/limitations.md]:
```suggestion
- No realtime API/subscriptions. In the future, we will have Prisma events engine (after the GA) that would pave way for such a feature, there is no ETA for this yet. 
```

**Comment by janpio** [comment]
Is this ready for merge or still being discussed?
Will this close #345?

**Comment by divyenduz** [comment]
From my end, it is ready to be merged.

**Pull request merged**